typos.toml

```diff
@@ -0,0 +1,10 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
```

Comment:
How about moving this file to the `.github/workflows` package? For example, create a new subdirectory.

Reply:
what dir name you suggest?

Reply:
`configs`?

Reply:
+1